bundles/org.openhab.binding.omnilink/src/main/resources/OH-INF/thing/unit.xml

```diff
@@ -198,6 +198,10 @@
 		<label>Unit Level</label>
 		<description>Increase/Decrease the level of this unit.</description>
 		<category>Slider</category>
+		<tags>
+			<tag>Control</tag>
+			<tag>Light</tag>
+		</tags>
 		<state min="0" max="100" pattern="%d %%"/>
 	</channel-type>
```

Comment:
Is this a light globe or a screen brightness control? See my long comment that shows how the lights appear on the location tabs, is that a desired result you want? I Don't have any of the devices from this binding to understand from reading the bindings documentation.

Reply:
Yes, an omnilink unit is traditionally a light switch that turns on or of a light, with dimmer functionality. There were a few non light switch units available but even when they were made they were rare.

Reply:
If you think it needs to be tagged then feel free to resolve this. The tags are only suggestions and can be changed at any time by the user, so if you think more people will benefit VS the number of people that see it as a bug and complain if the light counter is not accurate, I'm happy to agree with your assessment.